fetch.bs

```diff
@@ -6958,8 +6945,25 @@ the associated steps:
  <li><p>Let <var>promise</var> be <a>a promise resolved with</a> an empty
  <a for=/>byte sequence</a>.
 
- <li><p>If <var>object</var>'s <a for=Body>body</a> is non-null, then set <var>promise</var> to the
- result of <a>fully reading body as promise</a> given <var>object</var>'s <a for=Body>body</a>.
+ <li>
+  <p>If <var>object</var>'s <a for=Body>body</a> is non-null, then:
+
+  <ol>
+   <li><p>Set <var>promise</var> to <a>a new promise</a>.
+
+   <li><p>Let <var>fulfilledSteps</var> given a <a>byte sequence</a> <var>bytes</var> be to
```

Comment:
Same here. (Although I guess resolveSteps/rejectSteps would be OK too.)

Reply:
Done